opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -20,12 +20,29 @@
 from inspect import signature
 from typing import Optional
 
+import base64
+import PyPDF2
+import docx
+from pptx import Presentation
+from pylatexenc.latex2text import LatexNodes2Text
+from openai import OpenAI
+
 CURRENT_FILE = None
 CURRENT_LINE = 1
 WINDOW = 100
 
 ENABLE_AUTO_LINT = os.getenv('ENABLE_AUTO_LINT', 'false').lower() == 'true'
 
+# OPENAI
+OPENAI_API_KEY = os.getenv('OPENAI_API_KEY', '')
+OPENAI_BASE_URL = os.getenv('OPENAI_BASE_URL', 'https://api.openai.com/v1')
+OPENAI_MODEL = os.getenv('OPENAI_MODEL', 'gpt-4o-2024-05-13')
+MAX_TOKEN = os.getenv('MAX_TOKEN', 500)
+
+OPENAI_PROXY = f'{OPENAI_BASE_URL}/chat/completions'
+
+client = OpenAI(api_key=OPENAI_API_KEY, base_url=OPENAI_BASE_URL)
```

Comment:
Actually, can we add a check here: If `OPENAI_API_KEY` is NOT set, we remove the functions that requires `client` (`parse_audio`, `parse_image`, `parse_audio`) from the documentation?

Reply:
done